src/static/js/modules/init-chosen.js

```diff
@@ -10,7 +10,9 @@ require( 'chosen' );
 function init() {
   $( '.chosen-select' ).chosen( {
     width:           '100%',
+    /* eslint-disable */
```

Comment:
Not awesome in that you have to ignore the additional issues that come up, but at least the first rule being ignored is the real one.